runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/stableinput/BufferedElements.java

```diff
@@ -90,7 +90,8 @@ public int hashCode() {
 
     @Override
     public void processWith(DoFnRunner doFnRunner) {
-      doFnRunner.onTimer(timerId, timerFamilyId, window, timestamp, outputTimestamp, timeDomain);
```

Comment:
Can we pass the key through here?  We might need to include it in the coder for BufferedElements.Timer

Reply:
@reuvenlax  done